src/mapml.css

```diff
@@ -776,3 +776,45 @@ label.mapml-layer-item-toggle {
   ) {
   outline: 0!important;
 }
+
+
+/**
+* Feature Index
+ */
+.mapml-feature-index-box {
+  margin: -36px 0 0 -36px;
+  width: 72px;
+  height: 72px;
+  left: 50%;
+  top: 50%;
+  content: '';
+  display: block;
+  position: absolute;
+  z-index: 10000;
+}
```

Comment:
```suggestion
  z-index: 10000;
  outline: thin solid #fff;
}
```

Adding a thin outline to ensure the `mapml-feature-index-box` is visible independently of the background:

![outline-feature-index-box](https://user-images.githubusercontent.com/26493779/146523225-ffb9f114-92d3-44d0-8cc4-2255e6183df8.png)

(makes me wonder whether we should just do a black border with white outline instead of an SVG? But let's not, for now... 😋)